fastlane/lanes/release.rb

```diff
@@ -14,10 +14,46 @@
   #####################################################################################
   desc 'Creates a new release branch from the current trunk'
   lane :code_freeze do |options|
-    android_codefreeze_prechecks(skip_confirm: options[:skip_confirm])
+    # Ensure we use the latest version of the toolkit
+    check_for_toolkit_updates unless is_ci || ENV['FASTLANE_SKIP_TOOLKIT_UPDATE_CHECK']
 
-    android_bump_version_release()
-    new_version = android_get_app_version()
+    ensure_git_status_clean
+
+    Fastlane::Helper::GitHelper.checkout_and_pull(DEFAULT_BRANCH)
+
+    confirmation_message = <<-MESSAGE
+
+      Code Freeze:
+      • New release branch from #{DEFAULT_BRANCH}: release/#{next_release_version}
+      • Current release version and build code: #{current_release_version} (#{current_build_code}).
+      • New release version and build code: #{first_beta_version} (#{next_build_code}).
+
+      Do you want to continue?
+
+    MESSAGE
+
+    # Ask user confirmation
+    UI.user_error!('Aborted by user request') unless options[:skip_confirm] || UI.confirm(confirmation_message)
+
+    # Create the release branch
+    UI.message 'Creating release branch...'
+    ensure_git_branch(branch: DEFAULT_BRANCH)
+    Fastlane::Helper::GitHelper.create_branch("release/#{next_release_version}", from: DEFAULT_BRANCH)
+    UI.message "Done! New release branch is: #{git_branch}"
+
+    # Bump the version and build code
+    UI.message 'Bumping beta version and build code...'
+    Fastlane::Helper::GitHelper.ensure_on_branch!('release')
```

Comment:
Should this be the same as what we have on line 41?

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/19288/commits/0283d84ea2d2002df1bb8e56c6bebe3b3107b211